Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/ErrorLogServiceImpl.java

```diff
@@ -150,4 +152,15 @@ private static void logDebug(String msg, Object... objects) {
         }
     }
 
+    private static void addErrorMsgFrom(LogEventSimpleResponseMessageType response) {
+        if (hasError(response)) {
+            HelperUtil.addMessageError(null, response.getMessage());
+        }
+    }
+
+    private static boolean hasError(LogEventSimpleResponseMessageType response) {
+        return null != response && null != response.isStatus() && !response.isStatus()
```

Comment:
When code reaches this level, response never become null. If so, no need to check in here